server/api/person/person.controller.js

```diff
@@ -36,14 +38,28 @@ function listPeople (req, res) {
 
     Person.find(query, select).populate('tags').sort(sort)
       .then(got => {
-        res.json(got)
+        res.json(removeField(req, got))
       })
   } catch (e) {
     console.error('Bad request', req.query)
     return res.status(400).send(e)
   }
 }
 
+function removeField (request, queryResult) {
```

Comment:
This seems redundant - you are getting all the fields from the find call and then carefully removing the email field. 
Instead just add the fields you don't want to the select parameter of the find - prefix with a minus
Model.find({}, '-email, -phone',

Reply:
That might not be a good idea. Because user should be able to see all their fields. By adding parameter, user's email and phone number + other user email or phone will be filtered out. That might be not good if user look in to their own profile page and their phone number is hidden